src/UI/Implementation/Component/Panel/Factory.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation\Component\Panel;
+
+use ILIAS\UI\Component\Panel as P;
+
+/**
+ * Class Factory
```

Comment:
:-) 

It depends on how one works. In my case PHP Storms complains if a class does not provide a description by Doc Comment. An easy fix for this complaint is to auto generate one. This is kind of a mental hack since the problem is not visible anymore. But you are obviously right by pointing out that the issue of not having a description is actually still there.